lib/gtl_formatter.rb

```diff
@@ -177,9 +177,19 @@ def self.os_info_details(item)
     "svg/os-#{item.os_image_name}.svg"
   end
 
+  def self.item_provider(item)
+    type = item.class.to_s.split('::').last
+    case type
+    when 'ResourcePool', 'Container'
+      item.ext_management_system.decorate
+    else
+      item.decorate
```

Comment:
How are other types handled e.g. VMs / Hosts?  I don't see a `decorate` method on any of these in core, can we handle ResourcePools and Containers the same way?

Reply:
Host has this https://github.com/ManageIQ/manageiq-decorators/blob/master/app/decorators/host_decorator.rb

How do we connect `decorators` with classic?